app/views/content_page_versions/_form.html.erb

```diff
@@ -26,8 +26,7 @@
           <% if action_name === "edit" || action_name === "new" || action_name === "update" %>
             <div class="fixed-footer">
               <%= form.govuk_submit "Save as draft" %>
-              <%= link_to 'Cancel', edit_content_page_content_page_version_path(@content_page_version.content_page, @content_page_version), :data => {:confirm => "Are you sure you want to undo your changes ?", :class => "govuk-link"}%> |
-              <%= link_to 'Back', versions_content_page_path(@content_page_version.content_page), :data => {:confirm => "Any changes will be lost, are you sure ?", :class => "govuk-link"}%>
+              <%= link_to 'Cancel', versions_content_page_path(@content_page_version.content_page), data: { confirm: "Any changes will be lost, are you sure ?"}, class: "govuk-link" %>
```

Comment:
Really picky - but there should be a space at the end of the hash (before the closing curly bracket).
```suggestion
              <%= link_to 'Cancel', versions_content_page_path(@content_page_version.content_page), data: { confirm: "Any changes will be lost, are you sure ?" }, class: "govuk-link" %>
```